src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2.java

```diff
@@ -218,9 +220,17 @@ private static void runInspectAndDeidPipeline(
                         .withSideInputs(headers));
         break;
       case PARQUET:
+//        TODO: Remove KeyRange parameter, as it is unused
```

Comment:
Nit. Can you format the todo to align properly with below code?

Reply:
Noted! will update in the next PR.